metrics/metric.go

```diff
@@ -144,3 +140,12 @@ func ParseMetricName(name string) (string, []string, error) {
 
 	return name[0:openingTokenPos], tags, nil
 }
+
+// ObservedMetric is a static representation
+// of a tracked metric.
+type ObservedMetric struct {
+	*Metric
+	Sink       Sink
+	Thresholds []Threshold
```

Comment:
Do we need to tie thresholds to a particular metric? :thinking: This will prevent us from having thresholds that reference multiple metrics in the future (https://github.com/grafana/k6/issues/1441), and maybe even from improvements like https://github.com/grafana/k6/issues/1253